app/models/user.rb

```diff
@@ -13,7 +13,23 @@ def has_role?(role)
   end
 
   def role
-     roles&.first&._type.titleize
+    roles&.first&._type
+  end
+
+  def pending_submissions(user_id)
+    user = User.find(user_id)
+    user.submissions.where(form_id: 2, status: "pending")
+  end
```

Comment:
model methods must not be like that we should not us find in model methods this must be in controller